content/docs/get-started/gcp/begin.md

```diff
@@ -2,7 +2,7 @@
 title: Before You Begin | GCP
 h1: Before You Begin
 linktitle: Before You Begin
-meta_desc: This page provides an overview on how to get started with Pulumi when starting an GCP project.
+meta_desc: This page provides an overview on how to get started with Pulumi when starting an Google Cloud project.
```

Comment:
```suggestion
meta_desc: This page provides an overview on how to get started with Pulumi when starting a Google Cloud project.
```

Also, minor, but "get started .. when starting" reads a bit oddly.  Maybe just"Get started with Pulumi and Google Cloud"?